rust/feature-flags/src/flags/flag_operations.rs

```diff
@@ -212,6 +197,165 @@ mod tests {
     // TODO: Add more tests to validate deserialization of flags.
     // TODO: Also make sure old flag data is handled, or everything is migrated to new style in production
 
+    #[test]
```

Comment:
not in scope, but should we break out these tests to their own module too?  starting to get large.